src/utils/utils.cairo

```diff
@@ -514,4 +514,47 @@ namespace Helpers {
             return a;
         }
     }
+
+    // @notice convert bytes to little endian
+    func bytes_to_byte8_little_endian{range_check_ptr}(
```

Comment:
it may seem weird but it's `bytes8` in solidity (https://docs.soliditylang.org/en/v0.8.17/types.html#fixed-size-byte-arrays) and I'd suggest to keep the same naming convention

(they especially writes `bytes ~ bytes1[]` ..!)